main.py

```diff
@@ -8,20 +8,95 @@
 from shape import Rectangle, Circle, Triangle, Parallelogram, Shape
 from PIL import Image
 import glob
+from recording import SimpleRecording
+from subject import Round
+from subject import Instance
+from subject import Subject
+from datetime import date
+import time
 
 os.environ["SDL_VIDEO_CENTERED"] = "1"
 
+path = './IAPS/*.jpg'
+
+IAPS = []
+for filename in glob.glob(path):
+    IAPS.append(filename)
+
+
+def get_iap(file):
```

Comment:
I'm not sure why you needed to write this, are you just trying to get the name of the image? There are many handy ways to do this, https://www.pythonpool.com/python-get-filename-without-extension/

Failing that, you could put this as a for loop so you're not repeating your code like this